README.md

```diff
@@ -51,25 +51,18 @@ See [open issues](https://github.com/unjs/automd/issues?q=is%3Aopen+is%3Aissue+l
 
 ### `jsdocs` Generator
```

Comment:
Like my other suggestion about the "Arguments" heading, we have a "Generators" heading above this one so "`xyz` Generator" is sorta redundant in my opinion :)

```suggestion
### `jsdocs`
```